### PR TITLE
Thu/minor bug fixes

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

File: src/utils/SolutionsUtils.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ const patchSolutionIfLocalConfigExists = async (originalSolution) => {
 };
 
 const checkParametersValidationConstraintsInSolution = (data) => {
-  const parametersWithConstraint = data?.parameters?.filter((parameter) => parameter.options?.validation);
+  const parametersWithConstraint = data?.parameters?.filter((parameter) => parameter && parameter.options?.validation);
```
Comment (esasova) [inline src/utils/SolutionsUtils.js:65]: just a question: optional chaining operator only (parameter?.options?.validation) won't work in this case ? :slightly_smiling_face:

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: src/utils/SolutionsUtils.js (hunk 1 above)
Comment (csm-thu) [reply src/utils/SolutionsUtils.js]: Nice catch, I changed it to use the optional chaining operator. Thank you for the suggestion :pray:

**Review by nborde-CSM — APPROVED** [review]
LGTM :+1:

**Pull request merged**